lib/classes/ConfigSchemaHandler/resolveAjvValidate.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const Ajv = require('ajv').default;
+const objectHash = require('object-hash');
+const path = require('path');
+const os = require('os');
+const standaloneCode = require('ajv/dist/standalone').default;
+const fs = require('fs');
+const requireFromString = require('require-from-string');
+const fse = require('fs-extra');
+const deepSortObjectByKey = require('../../utils/deepSortObjectByKey');
+const ensureExists = require('../../utils/ensureExists');
+
+const getValidate = async (schema) => {
+  const cacheDir = path.resolve(os.homedir(), '.serverless/schemas');
+  const schemaHash = objectHash(deepSortObjectByKey(schema));
+  const cachePath = path.resolve(cacheDir, schemaHash);
+
+  const generate = async () => {
+    const ajv = new Ajv({
+      allErrors: true,
+      coerceTypes: 'array',
+      verbose: true,
+      strict: false,
+      code: { source: true },
+    });
+    require('ajv-keywords')(ajv, 'regexp');
+    require('ajv-formats').default(ajv);
+    const validate = ajv.compile(schema);
+    const moduleCode = standaloneCode(ajv, validate);
+    // As validation schemas are ~4 MB we should cleanup old ones before generating new one
+    await fse.emptyDir(cacheDir);
```

Comment:
I wouldn't do that. User may use two different services, one with plugin that extends schema, and other with one does not, and with above they will both clear its own resolved `validate`.

I wouldn't expect many of such schemas generated, so I'd leave it as that, and do not attempt any cleanup (we already generate custom resources that way, with no cleanup, and no complains so far)

Reply:
That's a very valid point :+1: I'm still a bit worried about these schemas growing, but I guess we can see if that causes any trouble for users and accommodate if needed, so I'm removing the cleanup step for now